examples/deployment/src/asa-helpers.js

```diff
@@ -0,0 +1,163 @@
+// Authors of example: Invertisment & amityadav0
+const algosdk = require('algosdk');
+
+// Function used to print created asset for account and assetid
+exports.printCreatedAsset = async function (deployer, account, assetid) {
+  // note: if you have an indexer instance available it is easier to just use this
+  //     let accountInfo = await indexerClient.searchAccounts()
+  //    .assetID(assetIndex).do();
+  // and in the loop below use this to extract the asset for a particular account
+  // accountInfo['accounts'][idx][account]);
+  let accountInfo = await deployer.algodClient.accountInformation(account).do();
+  for (idx = 0; idx < accountInfo['created-assets'].length; idx++) {
+    let scrutinizedAsset = accountInfo['created-assets'][idx];
```

Comment:
let's use `for (asset in accontInfo['created-assets'])` - this case you don't need to play with indexes.

Side note: you didn't want to use `forEach`? ;)

Reply:
This is mostly Amit's code so I added the pieces together without changing too much

Reply:
His code was for training, it was good, but now in the repo we can clean few bits.

Reply:
Changed